src/Kunstmaan/TranslatorBundle/Controller/TranslatorCommandController.php

```diff
@@ -74,6 +74,8 @@ public function importForcedAction()
      */
```

Comment:
Also add a `@deprecated` docblock with the same text as the deprecation for IDE completion